x-pack/plugins/infra/public/components/asset_details/context_providers.tsx

```diff
@@ -6,7 +6,7 @@
  */
 
 import React from 'react';
-import { AssetDetailsStateProvider } from './hooks/use_asset_details_state';
+import { AssetDetailsRenderPropsProvider } from './hooks/use_asset_details_render_props';
```

Comment:
Settled on `AssetDetailsRenderProps` name, seemed like it better conveys the purpose, but let me know if that does not look right 🙌

Reply:
nice!